### ISSUE TITLE
Accessibility: Settings Language Select Box does not show focus on "arrow"

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-ngx#6483]
dxp #699

**Comment by platon-rov** [comment]
There is nothing about NGX itself.

DXP/Jukebox was made to working as a web component in the DXP/Core, so it has its own shadow dom. Children of shadow dom cannot be included in the page's tab instead, the whole web component gets a focus. There is something similar about HTML's native video - the component has controls but you cannot reach them via keyboard (tab).

<img width="1466" alt="Screenshot 2021-09-26 at 12 59 15" src="https://user-images.githubusercontent.com/20265336/134803356-bc56fd11-d74f-493b-a58b-c54256dd7b6f.png">

Basically fix should be provided in DXP/Jukebox but if we're assuming the NGX components to be used as web components wee need to rewrite a lot :)
